spec/spec_helper.rb

```diff
@@ -2,7 +2,7 @@
 Bundler.setup
 
 begin
-  require 'pry-byebug'
+  require 'pry-byebug' if RUBY_VERSION <= '3.2'
```

Comment:
@noelrappin is pry-byebug incompatible with 3.2? 

I was checking their [gemspec](https://github.com/deivid-rodriguez/pry-byebug/blob/master/pry-byebug.gemspec), and it seems to be ok.

Reply:
I got this error

```
NameError:
  undefined method `=~' for class `Pry::Code'
```

Reply:
Might be a version thing, actually

Reply:
let me check

Reply:
The tricky part seems to be getting a version of pry and byebug that works for all the ruby versions

Reply:
Got it, the problem was with the version of pry-byebug as you suggested. Great catch!